src/Raven.Server/Web/Operations/OperationsHandler.cs

```diff
@@ -22,7 +22,10 @@ public Task GetNextOperationId()
             {
                 using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
                 {
+                    writer.WriteStartObject();
+                    writer.WritePropertyName("id");
```

Comment:
It will break studio, In getNextOperationId add 4-th parameter: x => x.Id (assuming you use Id with upper case letter -which is by convention). So line 12 should be:
```
return this.query(endpoints.databases.operations.operationsNextOperationId, null, this.db, x => x.Id);
```